src/main/java/com/cozyhome/onlineshop/userservice/controller/UserSecuredController.java

```diff
@@ -63,7 +64,19 @@ public ResponseEntity<UserInformationResponse> updateUserInfo(@RequestBody @Vali
         return ResponseEntity.ok(userService.updateUserData(userInformationDto, userId));
     }
 
-    @Operation(summary = "Get user information")
+    @Operation(summary = "Update user password in personal account")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = SwaggerResponse.Code.CODE_200, description = SwaggerResponse.Message.CODE_200_FOUND_DESCRIPTION) })
+    @Secured({"ROLE_CUSTOMER"})
```

Comment:
propose to add "ROLE_MANAGER", "ROLE_ADMIN", admin and manager can also have personal account and should have ability to change personal data